dotcom-rendering/src/web/components/SignInGate/gateDesigns/SignInGateMainCheckoutComplete.tsx

```diff
@@ -0,0 +1,202 @@
+import { cmp } from '@guardian/consent-management-platform';
+import { Button, Link, LinkButton } from '@guardian/source-react-components';
+import { trackLink } from '../componentEventTracking';
+import type {
+	CheckoutCompleteCookieData,
+	Product,
+	SignInGatePropsWithCheckoutCompleteCookieData,
+	UserType,
+} from '../types';
+import {
+	actionButtons,
+	bodyBold,
+	bodySeparator,
+	bodyText,
+	faq,
+	firstParagraphOverlay,
+	headingStyles,
+	hideElementsCss,
+	laterButton,
+	privacyLink,
+	registerButton,
+	signInGateContainer,
+	signInHeader,
+	signInLink,
+} from './shared';
+
+const getHeadingText: (userType: UserType) => string = (userType) => {
+	const headingMap: Record<UserType, string> = {
+		new: 'Complete your registration',
```

Comment:
Wondering whether these strings should be stored as a `const` in a different file?

Reply:
Yeah, just thinking if we stored the string in one const and the copy changed we'd only need to update one variable, rather than the few times we use it in the implementation and in the test. 
Could keep it in the same file, my main thought was whether it should be assigned to one variable rather than repeated

Reply:
oh, yes. let's do that.

Reply:
Ok cool, I've started doing this on a separate branch: https://github.com/guardian/dotcom-rendering/tree/ld-vb-sign-after-checkout-consts 

At the moment importing the consts into the tests file is breaking the tests, I'll dig in to in on Monday. Have a good weekend

Reply:
Made variables within the file now.